cpu/stm32f4/ldscripts/stm32f413vg.ld

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32f4
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32F413VG
+ *
+ * @author          Vincent Dupont <vincent@otakeys.com>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x08000000, LENGTH = 1024K
+    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 256K
```

Comment:
According to the datasheet, the values here don't seem to be valid. I think it should be:
```
rom (rx)    : ORIGIN = 0x08000000, LENGTH = 1.5M (not sure about the dot)
ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 320K
cpuid (r)   : ORIGIN = 0x1fff7a10, LENGTH = 12
```
and there's no ccmram apparently.